src/Altinn.Apps/AppFrontend/react/altinn-app-frontend/src/App.tsx

```diff
@@ -20,7 +20,10 @@ const TEN_MINUTE_IN_MILLISECONDS: number = 60000 * 10;
 export const App = () => {
   const dispatch = useAppDispatch();
   const hasErrorSelector = makeGetHasErrorsSelector();
-  const hasApiErrors: boolean = useAppSelector(hasErrorSelector);
+    const hasApiErrors: boolean = useAppSelector(hasErrorSelector);
+    // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+    // @ts-ignore since appliacationSetting is not available in rootstate
+    const appOidcProvider = useSelector((state) => state.applicationSettings?.applicationSettings?.appOidcProvider);
```

Comment:
@lorang92 @haakemon  is this correct the merge. UseSelector?

Reply:
No, it should be `useAppSelector`, and that also means the two comments above can finally be removed :)